### PR TITLE
[ContactStructuralMEchanicsApplication]Clean up table stream (transition PR)

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

**Comment by philbucher** [comment]
Could you post a screenshot of how the colors look?

**Comment by loumalouomega** [comment]
It is exactly the same as the current output. This is a transition PR. In the next one I will replace for the table logger

**Comment by philbucher** [comment]
I am really looking forward to this
Thx!

File: kratos/python/add_utilities_to_python.cpp [hunk 1]
```diff
@@ -57,6 +57,18 @@ namespace Kratos
 namespace Python
 {
 
+/**
+ * @brief Sets the current table utility on the process info
+ * @param rCurrentProcessInfo The process info
+ */
+void SetOnProcessInfo(
```
Comment (philbucher) [inline kratos/python/add_utilities_to_python.cpp:64]: why do you need this function?
Isn't it easier to do it directly?

**Review by philbucher — COMMENTED** [review]
Looks ok to me, but the name Table might be misleading since there is already a table in the core

File: kratos/python/add_utilities_to_python.cpp (hunk 1 above)
Comment (loumalouomega) [reply kratos/python/add_utilities_to_python.cpp:64]: Yes, but it can be done right now due to #2033

**Comment by loumalouomega** [comment]
table stream already exists, this is just a transition PR (I don't want to do it all at once)

**Comment by philbucher** [comment]
ok, fine for me

I will approve on Tuesday to give others also time to have a look

**Comment by philbucher** [comment]
Btw are you done modifying the TableStream utility or will there be another PR?

I am asking bcs I would also use it

File: kratos/includes/table_stream.h [hunk 2]
```diff
@@ -15,16 +15,23 @@
 #ifndef KRATOS_TABLE_STREAM_H_INCLUDED
```
Comment (philbucher) [inline kratos/includes/table_stream.h:15]: there are some comments in the header of this file, I think at least some of them are outdated

Comment (loumalouomega) [reply kratos/includes/table_stream.h:15]: :grin: Maybe, but this class will disappear eventually

**Comment by loumalouomega** [comment]
@philbucher Before starting with the new developments related with the table logger I need to merge this

**Review by philbucher — APPROVED** [review]
sorry I forgot to approve yesterday

**Comment by loumalouomega** [comment]
Thanks

**Pull request merged**